Tests/FunctionalTests/PluginTests.swift

```diff
@@ -263,9 +293,11 @@ class PluginTests: XCTestCase {
 
             // Load the package graph.
             let packageGraph = try workspace.loadPackageGraph(rootInput: rootInput, observabilityScope: observability.topScope)
-            XCTAssert(observability.diagnostics.isEmpty, "\(observability.diagnostics)")
-            XCTAssert(packageGraph.packages.count == 1, "\(packageGraph.packages)")
-            let package = try XCTUnwrap(packageGraph.packages.first)
+            XCTAssert(!observability.hasErrorDiagnostics, "\(observability.diagnostics)")
+            XCTAssert(!observability.hasWarningDiagnostics, "\(observability.diagnostics)")
```

Comment:
there is also XCTAssertNoDiangostics

Reply:
Thanks, didn't know about that one.  The problem here was that I don't actually care about info and debug diagnostics (there might be some of those).  I saw the `problemsOnly` flag to XCTAssertNoDiags, that's great.

Reply:
In this case we get things like `- [debug]: [DependencyResolver] solved 'mypackage'`

Reply:
Changed it